mod/wikirate_project/data/real.yml

```diff
@@ -1054,6 +1059,18 @@
   codename: :social_description
   content: "Wikirate International is the non-profit that hosts the Wikirate platform. We empower people and organizations to collect data and develop insights on companies under a free license."
 
+- name: "Events Description"
+  type: :basic
+  codename: :events_description
+  content: "Here you can find all upcoming and past events."
+
+- :name: "Image on Socials"
+  :type: :image
+  :codename: :image_on_socials
+  :image: image_on_socials.png
+  :storage_type: :coded
```

Comment:
fyi, in concept `coded` images are supposed to be for images bound to the mod code. As in, the image should be stored in the code, not in the cloud.  I'm not convinced it's all working properly, but you can probably skip that here.

Reply:
I removed the storage_type field from the card